docs/graphql/core/migrations/config-v1/advanced/auto-apply-migrations.rst

```diff
@@ -50,7 +50,7 @@ Example:
    # Start Hasura after applying the migrations present in /home/me/my-project/migrations
    docker run -p 8080:8080 \
           -v /home/me/my-project/migrations:/hasura-migrations \
-          -e HASURA_GRAPHQL_DATABASE_URL=postgres://postgres:@postgres:5432/postgres \
+          -e HASURA_GRAPHQL_DATABASE_URL=postgres://<postgres>:@<postgres>:5432/<postgres> \
```

Comment:
This is a specific example that users should just be able to copy. We don't need placeholders here.

Reply:
True, I was sceptical too